### PR TITLE
Add support for docker secrets for sensitive values

### PR DESCRIPTION
**Pull request by mingan666**
Fix for #62 

Adding support for the following alternative variables:

* DB_PASSWORD_FILE
* REDIS_PASSWORD_FILE
* MAIL_PASSWORD_FILE
* PUSHER_APP_KEY_FILE
* PUSHER_APP_SECRET_FILE

Not sure how to test the PUSHER variables, since they do not seem to be present in the .env file.

The _FILE variables are only checked if the original variable is not set, so if both are set the original variable will be used.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -93,6 +93,12 @@ Change the environment variables in the [provided example](./docker-compose.yml)
 
 Note that in order to avoid writing credentials directly into the file, you can create a `db_secrets.env` and use the `env_file` directive (see the [docs](https://docs.docker.com/compose/environment-variables/#the-env_file-configuration-option)).
 
+### Docker secrets
+
+As an alternative to passing sensitive information via environment variables, _FILE may be appended to some of the environment variables, causing the initialization script to load the values for those variables from files present in the container. In particular, this can be used to load passwords from Docker secrets stored in /run/secrets/<secret_name> files.
```
Comment (d7415) [inline README.md:98]: As there are only 5 of them, I think it's better to specify them

File: .gitignore
Comment (d7415) [inline .gitignore:1]: I'm against adding gitignore lines for IDE-specific files, as that could get out of hand quickly.

**Review by d7415 — COMMENTED** [review]
Thanks! I haven't tested it, but hopefully you have, and it looks like it should work.

File: README.md (hunk 1 above)
Comment (mingan666) [reply README.md:98]: added a list supported _FILE variables

File: .gitignore
Comment (mingan666) [reply .gitignore:1]: Removed

**Comment by mingan666** [comment]
I've tested most of the combinations - with original, with _FILE and with both.

However, as I wrote above, the PUSHER variables do not seem to be in the generated .env-file so not sure how to test those?

**Comment by d7415** [comment]
> 

Yeah, you're right. These were removed from `.env.example` in the main repo, so the replacements will no longer work.

I've removed those (and a few others) from the code.

**Review by d7415 — APPROVED** [review]

**Pull request merged**